docs/resources/metastore_data_access.md

```diff
@@ -53,6 +53,28 @@ resource "databricks_metastore_data_access" "this" {
 }
 ```
 
+For Azure using managed identity as credential (Private Preview)
```

Comment:
Did you get a green light from @annashres to document it publicly for private preview? 

If not - please remove this section from registry docs just yet, create a google doc in GTM private preview folder and move it there. And also link it on internal wiki for tf provider and azmi preview.

Reply:
Anna said it should be fine since the UC is gated as well and we'll ungate MIs when the UC is ungated

Reply:
Then it goes to this release :)